### DISCUSSION TITLE
Additional field types for Categories

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#2965] [Feature Requests]
I find the options available for Category fields very limiting, as things stand. More options for fieldtypes are really needed to make the Categories interface more useful and usable.
At the moment, I'm sometimes having create a parallel Channel, who's Entries are then mapped with each Category, to supply the additional Fieldtypes - and give Content Editors a more presentable content editing experience in the CP.

It would be great if editing Category data / content, worked in the same way as editing Entries - including having the same control over field Layouts in the CP, as per Channel Entries.

Also in relation to the above, it would be great if we could make the current native Category Description field an RTE field.

**Comment by paulcripps** [comment]
I agree with the above and would add it would be great to be able to add further functionality to allow us to set categories so that users can only select a single category.

I'd also add the ability to make categories on a group by group basis required.

**Comment by robsonsobral** [comment]
And freeing the category groups to be used across sites, in case needed.
